tests/py/dynamo/converters/test_neg_aten.py

```diff
@@ -0,0 +1,24 @@
+import torch
+import torch.nn as nn
+from parameterized import parameterized
+from torch.testing._internal.common_utils import run_tests
+from torch_tensorrt.dynamo.test_utils import DispatchTestCase
+from torch_tensorrt import Input
+
+
+class TestNegConverter(DispatchTestCase):
+    def test_neg(self):
+        class neg(nn.Module):
+            def forward(self, input):
+                return torch.neg(input)
+
+        inputs = [torch.randn(1, 10)]
```

Comment:
I believe `randn` defaults to `torch.int64`, so consider switching to `rand` for float or adding `.int()` for int, to the instantiation of the tensor

Reply:
Hey @gs-olive I went through the pytorch documentation here: https://pytorch.org/docs/stable/generated/torch.randn.html
I don't think it's `torch.int64` because in the examples we see float output tensor.

Reply:
I believe `randn` should cover the different data types. Also you could try including the `parameterized` test cases to check for different shapes of the input tensort.